src/main/java/uk/gov/hmcts/reform/prl/services/c100respondentsolicitor/validators/RespondentMiamChecker.java

```diff
@@ -42,21 +42,21 @@ public boolean isStarted(PartyDetails respondingParty) {
     @Override
     public boolean isFinished(PartyDetails respondingParty) {
         Optional<Response> response = findResponse(respondingParty);
-
+        boolean isFinished = false;
```

Comment:
can this be consistent with the previous change? boolean will be false by default

Reply:
@shashisk97 this is a local variable inside inside method so won't be initialised with default value , previous changes this variable is getting initialised in different flows so we don't need to initialise it there